src/components/App.js

```diff
@@ -1,8 +1,196 @@
+import React, { useState } from 'react';
+import SelectNumber from './Select';
+import Board from './Board';
+import { alphabet } from '../letters/Alphabet';
+import { wordDictionary2 } from '../letters/WordDictionary2';
+import { wordDictionary3 } from '../letters/WordDictionary3';
+import { wordDictionary4 } from '../letters/WordDictionary4';
+import { wordDictionary5 } from '../letters/WordDictionary5';
+import { letterPoints } from '../letters/LetterPoints';
 
 function App() {
+  const [selectNumber, setSelectNumber] = useState(false);
+  const [wordNo, setWordNo] = useState(1);
+  const [numberSelected, setNumberSelected] = useState('');
```

Comment:
I think selectNumber/numberSelected need better names, its hard to distinguish what they are compared to eachother

Reply:
Done. numberSelected changed to maxNumberConsonants and selectedNumber to showBoard.